contracts/ColonyStorage.sol

```diff
@@ -31,59 +31,59 @@ contract ColonyStorage is CommonStorage, ColonyDataTypes, DSMath {
   // this one will have the getters. Make custom getters in the contract that seems most appropriate,
   // and add it to IColony.sol
 
-  address colonyNetworkAddress;
-  ERC20Extended token;
-  uint256 rewardInverse;
+  // Colony-wide roles
+  uint8 constant FOUNDER_ROLE = 0;
```

Comment:
We c o u l d be using enums here instead... #semantecs

But I'll survive :)

Reply:
Tbh I need to give `enum`s another chance. It's a brand new year after all.. I've been biased against them for a while and I can no longer remember why.

Reply:
There we go, enums all around. See how you like that @kronosapiens